lib/ret/media_search.ex

```diff
@@ -216,7 +242,8 @@ defmodule Ret.MediaSearch do
         )
 
       res =
-        "https://api.twitch.tv/kraken/search/streams?#{query}" |> retry_get_until_success([{"Client-ID", client_id}])
+        "https://api.twitch.tv/kraken/search/streams?#{query}"
+        |> retry_get_until_success([{"Client-ID", client_id}])
```

Comment:
Is your editor not respecting the 120 line length in the `.formatter.exs` file?

Reply:
ah must not be. will take a look as to why not

Reply:
Yep that was the issue